modules/apps/transfer/keeper/grpc_query_test.go

```diff
@@ -245,12 +245,24 @@ func (suite *KeeperTestSuite) TestEscrowAddress() {
 			},
 			true,
 		},
+		{
+			"failure - channel not found",
+			func() {
+				req = &types.QueryEscrowAddressRequest{
+					PortId:    ibctesting.InvalidID,
+					ChannelId: ibctesting.FirstChannelID,
+				}
+			},
+			false,
+		},
 	}
 
 	for _, tc := range testCases {
 		tc := tc
 		suite.Run(fmt.Sprintf("Case %s", tc.msg), func() {
 			suite.SetupTest() // reset
+			path := ibctesting.NewTransferPath(suite.chainA, suite.chainB)
```

Comment:
I added this part to enable creation of the channel, otherwise the test for the "happy path" would fail.